src/doc/reference.md

```diff
@@ -2319,6 +2337,8 @@ impl<T: PartialEq> PartialEq for Foo<T> {
 }
 ```
 
+You can implement derive for your own type through [procedural macros](#procedural-macros).
```

Comment:
Earlier in this section, "derive" is styled as

```
`derive`
```

with backticks, so might be good to be consistent.